admin/client/src/legacy/AddToCampaignForm.js

```diff
@@ -9,7 +9,7 @@ $.entwine('ss', function($){
             if (dialog.length) {
                 dialog.open();
             } else {
-                dialog = $('<div id="add-to-campaign__dialog" />');
+                dialog = $('<div id="add-to-campaign__dialog" class="add-to-campaign__dialog" />');
```

Comment:
Is this in the BEM style guides somewhere? If there's only ever one of anything I don't see the problem (given it's attached to body and never removed, we _want_ there to only be one, not an ever-increasing number). But if it's against style, sure.